Vagrantfile

```diff
@@ -68,6 +68,7 @@ Vagrant.configure(2) do |config|
           "AWS_ACCESS_SECRET": ENV["AWS_ACCESS_SECRET"],
           "AWS_REGION": ENV["AWS_REGION"],
           "PERM_ENV": "local",
+          "PERM_HOSTNAME": "localdev",
```

Comment:
Hm, I don't follow how this works; where is `PERM_HOSTNAME` used? I don't see it in `infrastructure` or in this repo.

Reply:
This was from an older iteration of a change that I made to infrastructure, and is, in fact, no longer needed. Thanks for the keen eye! Will revert this and the change below, and change the PR title.